requirements.txt

```diff
@@ -1,2 +0,0 @@
-boto3==1.7.59
-peewee==3.6.4
```

Comment:
Don't we need the package anymore?

Reply:
We don't need the packages in the root folder. We need them in the chalice folder so that they get imported into the lambda package built by chalice deploy.